### PR TITLE
Add weekly update 2016-11-04

### PR DESCRIPTION
**Pull request by JungMinu**
Please review this post. :heart:,@nodejs/evangelism

### CODE REVIEW

**Review by ryanmurakami — APPROVED** [review]

**Comment by Knighton910** [comment]
LGTM 🐧

File: locale/en/blog/weekly-updates/weekly-update.2016-11-04.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+date: 2016-11-04T12:00:00.000Z
+status: publish
+category: weekly
+slug: weekly-update-2016-11-04
+layout: blog-post.hbs
+---
+
+### Node v7.1.0 proposal
+
+* [Node v7.1.0 proposal](https://github.com/nodejs/node/pull/9438). The complete changelog from previous releases can be found [on GitHub](https://github.com/nodejs/node/blob/master/CHANGELOG.md).
```
Comment (Trott) [inline locale/en/blog/weekly-updates/weekly-update.2016-11-04.md]: Nit: I would leave out `from previous releases` and just write `The complete changelog can be found...`

File: locale/en/blog/weekly-updates/weekly-update.2016-11-04.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+### Community Updates
+
+* [Node.js v7 Release Brief](https://nodesource.com/blog/node-js-v7-release-brief)
+* [Last Week in Node.js Working Groups](https://nodesource.com/blog/last-week-in-node-js-working-groups-october-17-24): Last week was quieter than usual as a lot of effort was put into the new v7 Current release line, as such, this is the summary for the last two weeks.
```
Comment (Trott) [inline locale/en/blog/weekly-updates/weekly-update.2016-11-04.md]: Nit: Fix comma splice: `...Current release line. As such, this is...`

**Comment by Trott** [comment]
Some small nits, but LGTM with or without them corrected.

**Comment by JungMinu** [comment]
@Trott All nits addressed and updated :)

**Pull request merged**